### DISCUSSION TITLE
How to control sizing of components and the text inside them.

### DISCUSSION THREAD

**Discussion by julie777** [quasarframework/quasar#10767] [General - Components / Directives / etc]
I am relatively new to quasar and the defaults for the components seem to be too tall for the text or icons inside them. You can see in the clipping that my list item is at least 4 times as tall as the text in the input.

Here is an example from a [codepen](https://codepen.io/Julie777/pen/ExXRWvp) I did.
![image](https://user-images.githubusercontent.com/5090005/134220930-433644ec-38b7-45d5-9068-fc5e5fc7cecd.png)

When I inspect the elements in the browser I don't see any margins or padding in the CSS to account for the extra space.
I added a couple paragraphs to the codepen and font size and line spacing look good and there is space add after a paragraph to separate it from the next one. 

I think my list items look terrible and hope someone can help me understand how to fix them.

**Comment by pdanpdan** [comment]
You have dense to decrease the vertical size.
You can use some other display design (outline, borderless, ...)
For something else you need to style it with css
